service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -179,6 +162,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(caseDetails.getData())
+            .errors(orderValidationService.validate(updated))
```

Comment:
I would wrap the above code in an if that means that we don't have to do any additional computation if there are errors